packages/catalog-process/kubernetes/dev/configmap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: interop-be-refactor-catalog-process
+  namespace: dev-refactor
+data:
+  READMODEL_DB_HOST: "interop-read-model-dev.cluster-c9zr6t2swdpb.eu-central-1.docdb.amazonaws.com"
+  READMODEL_DB_NAME: "read-model-refactor"
+  READMODEL_DB_PORT: 27017
+  EVENTSTORE_DB_HOST: "interop-persistence-management-dev.cluster-c9zr6t2swdpb.eu-central-1.rds.amazonaws.com"
+  EVENTSTORE_DB_NAME: "persistence_management_refactor"
+  EVENTSTORE_DB_PORT: 6001
```

Comment:
Maybe this kind of data should go into the commons configmap to avoid duplication, what do you think? @Viktor-K

Reply:
IMHO `/packages/common` represents a separate package with its own env variables that potentially may change independently from other packages; however, we can consider having a `/kubernetes` folder for all common configurations at top level.

Reply:
I think there should be a common configmap somewhere to avoid duplicating shared data. Let's go with your idea 👍